annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -398,16 +389,6 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
         return isoformOverrideService.getOverrideSources();
     }
 
-    @ApiOperation(value = "Gets a list of available isoform override data sources",
-        nickname = "postIsoformOverrideSources")
-    @RequestMapping(value = "/isoform_override/sources",
```

Comment:
same comment as previous (maybe this should become a service if it is not an endpoint)

Reply:
A separate internal `IsoformOverrideController` is on the way.